src/index.html

```diff
@@ -1,19 +1,59 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
+  <meta name="viewport" content="width=device-width,
           user-scalable=no,
           initial-scale=1.0,
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
-</head>
-<body>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
+  </head>
 
-<script type="text/javascript" src="./main.js"></script>
+<body>
+  <div class="wrapper" data-qa="card">
+    <div class="wrapper__img">
+      <img src="./images/imac.jpeg" alt="imac">
+    </div>
+    <div class="wrapper__discription">
+      Моноблок APPLE A1419 iMac
+      27" Retina 5K (MNED2UA/A)
+    </div>
+    <div class="wrapper__code">Код товара: 195434
+    </div>
+    <div class="wrapper__rating">
+      <div class="wrapper__rating__group">
+        <img src="./images/star-active.svg" alt="">
```

Comment:
This is bed idea to use tag img here. Because if you need change style for active to non-active so you need to manipulate with HTML, batter use css class which will be responsible for active and non-active styles.